test/unit/search/dream_index/binning_directory_test.cpp

```diff
@@ -0,0 +1,265 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/search/dream_index/binning_directory.hpp>
+#include <seqan3/search/dream_index/concept.hpp>
+#include <seqan3/test/cereal.hpp>
+#include <seqan3/test/pretty_printing.hpp>
+
+using namespace seqan3;
+
+template <typename T>
+class binning_directory_test : public ::testing::Test
+{};
+
+template <typename T>
+class binning_directory_compressed_test : public ::testing::Test
+{};
+
+using binning_directory_types = ::testing::Types<binning_directory>;
+using binning_directory_compressed_types = ::testing::Types<binning_directory_compressed>;
+
+TYPED_TEST_CASE(binning_directory_test, binning_directory_types);
+TYPED_TEST_CASE(binning_directory_compressed_test, binning_directory_compressed_types);
```

Comment:
what are you doing here? o.O
A typed test should be used with multiple types, otherwise, there is no point for a typed test. 
But you copy and pasted all tests for two typed test each testing only one type...?

Reply:
It's from earlier and not refactored. I'll just do fixtures

Reply:
still open